pkgs/servers/mx-puppet-discord/generate.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env nix-shell
```

Comment:
Why are we not using the global nodePackages?

Reply:
Had a go at doing this, but building it straight from nodePackages complains about missing node-pre-gyp, and overriding that adding it as a build input has mismatched versions of some dependencies, so it hasn't taken into account the lockfile. In short, I'm unable to get this working with the global nodePackages.

Reply:
I think this is fine for leaf packages (i.e. applications). Thanks for trying with the global nodePackages set, anyway.